images/hub/requirements.txt

```diff
@@ -1,29 +1,78 @@
-# JupyterHub's version is defined in chartpress.yaml
-
-## Authenticators
-jupyterhub-dummyauthenticator==0.3.*
-jupyterhub-firstuseauthenticator==0.12.*
-jupyterhub-tmpauthenticator==0.6.*
-jupyterhub-ltiauthenticator==0.4.*
-jupyterhub-ldapauthenticator==1.2.*
-jupyterhub-hmacauthenticator==0.1.*
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile
+#
+alembic==1.3.3            # via jupyterhub
+async-generator==1.10     # via jupyterhub, jupyterhub-kubespawner
+attrs==19.3.0             # via jsonschema
+bcrypt==3.1.7             # via jupyterhub-firstuseauthenticator
+cachetools==4.0.0         # via google-auth
+certifi==2019.11.28       # via kubernetes, requests
+certipy==0.1.3            # via jupyterhub
+cffi==1.13.2              # via bcrypt, cryptography
+chardet==3.0.4            # via requests
+click==7.0                # via pip-tools
+cryptography==2.8         # via pyjwt, pyopenssl
+decorator==4.4.1          # via traitlets
+entrypoints==0.3          # via jupyterhub
+escapism==1.0.0           # via jupyterhub-kubespawner
+globus-sdk[jwt]==1.8.0
+google-auth==1.11.0       # via kubernetes
+idna==2.8                 # via requests
+importlib-metadata==1.5.0  # via jsonschema
+ipython-genutils==0.2.0   # via traitlets
+jinja2==2.11.1            # via jupyterhub, jupyterhub-kubespawner
+jsonschema==3.2.0         # via jupyter-telemetry
+jupyter-telemetry==0.0.4  # via jupyterhub
+jupyterhub-dummyauthenticator==0.3.1
+jupyterhub-firstuseauthenticator==0.13.0
+jupyterhub-hmacauthenticator==0.1
+jupyterhub-kubespawner==0.11.1
+jupyterhub-ldapauthenticator==1.2.2
```

Comment:
I'd like ldapauthenticator 1.3.0 be part of beta.4 that is now out, so perhaps refreeze -> get this merged and -> release beta.4?